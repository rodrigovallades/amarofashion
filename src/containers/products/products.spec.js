import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import { Products } from './index'

describe('Products container', function () {

  it('renders "Products" deep correctly', () => {
    const tree = renderer.create(<Products/>);
    expect(tree).toMatchSnapshot();
  });

  it('renders "Products" shallow correctly', () => {
    const component = shallow(<Products/>);
    expect(component.exists()).toEqual(true);
  });
})
