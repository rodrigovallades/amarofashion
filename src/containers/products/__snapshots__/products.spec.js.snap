// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Products container renders "Products" deep correctly 1`] = `
<div>
  <h1
    className="app__title"
  >
    Products (
    0
    )
  </h1>
  <div
    className="products"
  />
</div>
`;
